src/test/csf/Peer.h

```diff
@@ -355,9 +355,15 @@ struct Peer
     */
 
     bool
-    connect(Peer & o, SimDuration dur)
+    connect(Peer & o, SimDuration const & delay)
     {
-        return net.connect(this, &o, dur);
+        return connect(o,
+                DurationDistributionRef{ConstantDuration{delay}});
```

Comment:
If `DurationDistributionRef` takes its argument by reference, won't that reference be dangling since `ConstantDuration{Delay}` is a local temporary?